net/lp_rpc.proto

```diff
@@ -368,6 +376,37 @@ message NotifySegment {
     reserved 33; // Formerly "repeated VideoProfile fullProfiles"
 }
 
+// Sent by the aiworker to register itself to the orchestrator.
+message RegisterAIWorkerRequest {
+
+    // Shared secret for auth
+    string secret = 1;
+
+    // AIWorker capacity
+    int64 capacity = 2;
```

Comment:
Correct, it is not used. I was hesitent to remove it until we got farther through review in case there can be a use for it in addition to managing capacity per model.

Reply:
If it's not used, then please remove it. It's always simple to add a new value to proto def if needed.